### PR TITLE
[EDU-2471] Add RE missing variables

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

Added the following variables (unlisted in UI but accepted by the code):
- remote_port
- server_addr
- server_port
- tcpinfo_rtt

[EDU-2471]: https://aziontech.atlassian.net/browse/EDU-2471?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/en/pages/products/edge-application/rules-engine.mdx [hunk 1]
```diff
@@ -94,18 +94,22 @@ import TableContainer from "~/components/Table/TableContainer.astro"
...
 | `${request}` | Complete request query. | Request Phase<br />Response Phase |
 | `${request_method}` | The HTTP method of the request. For example: `GET`. | Request Phase<br />Response Phase |
 | `${request_uri}` | The complete URI of the request, with arguments (query string). | Request Phase<br />Response Phase |
 | `${scheme}` | The scheme of the request. For example: `http` or `https`. | Request Phase<br />Response Phase |
-| `${sent_http_name}` | The value of the response header field name. The name argument must be converted to lowercase and the hyphens must be converted to underscore. For example: `${sent_http_content_length}` will take the value of the Content-Length header. | Response Phase |
+| `${sent_http_name}` | The value of the response header field name. The name argument must be converted to lowercase and the hyphens must be converted to underscore. For example: `${sent_http_content_length}` will take the value of the `Content-Length` header. | Response Phase |
+| `${server_addr}` | The IP address of the server that is receiving the HTTP request. | Request Phase |
+| `${server_port}` | The HTTP port of the server that receives the request. | Request Phase |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/products/edge-application/rules-engine.mdx:110]: can it be "is receiving", same as the one on top?

File: src/content/docs/en/pages/products/edge-application/rules-engine.mdx [hunk 2]
```diff
@@ -94,18 +94,22 @@ import TableContainer from "~/components/Table/TableContainer.astro"
...
 | `${status}` | Status code of the response. | Response Phase |
+| `${tcpinfo_rtt}` | Round Trip Time (RTT) of the client's TCP connection, in microseconds. | Response Phase |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/products/edge-application/rules-engine.mdx:112]:
```suggestion
| `${tcpinfo_rtt}` | Round-Trip Time (RTT) of the client's TCP connection, in microseconds. | Response Phase |
```

File: src/content/docs/pt-br/pages/produtos/edge-application/rules-engine/index.mdx [hunk 3]
```diff
@@ -92,23 +94,28 @@ Por exemplo, o seguinte critério identifica se um usuário está usando um nave
...
 | `${request_uri}` | A URI completa da requisição, com argumentos (query string). | Request Phase <br />Response Phase |
 | `${scheme}` | O scheme da requisição: `http` ou `https`. | Request Phase <br />Response Phase |
-| `${sent_http_name}` | O valor do campo de cabeçalho de resposta name. O argumento name deve ser convertido para letras minúsculas e os hífens devem ser convertidos para underscore. Por exemplo: `${sent_http_content_length}` assumirá o valor do cabeçalho Content-Length. | Response Phase |
+| `${sent_http_name}` | O valor do campo de cabeçalho de resposta name. O argumento name deve ser convertido para letras minúsculas e os hífens devem ser convertidos para underscore. Por exemplo: `${sent_http_content_length}` assumirá o valor do cabeçalho `Content-Length`. | Response Phase |
+| `${server_addr}` | O endereço IP do servidor que está recebendo a requisição HTTP. | Request Phase |
+| `${server_port}` | A porta HTTP do servidor que recebe a requisição. | Request Phase |
 | `${status}` | O status code da resposta. | Response Phase |
+| `${tcpinfo_rtt}` | Round Trip Time (RTT) da conexão TCP do cliente, em microsegundos. | Response Phase |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/produtos/edge-application/rules-engine/index.mdx:112]:
```suggestion
| `${tcpinfo_rtt}` | Round-Trip Time (RTT) da conexão TCP do cliente, em microsegundos. | Response Phase |
```

File: src/content/docs/pt-br/pages/produtos/edge-application/rules-engine/index.mdx [hunk 4]
```diff
@@ -92,23 +94,28 @@ Por exemplo, o seguinte critério identifica se um usuário está usando um nave
...
 | `${request_method}` | O método HTTP da requisição. Por exemplo: `GET`. | Request Phase <br />Response Phase |
 | `${request_uri}` | A URI completa da requisição, com argumentos (query string). | Request Phase <br />Response Phase |
 | `${scheme}` | O scheme da requisição: `http` ou `https`. | Request Phase <br />Response Phase |
-| `${sent_http_name}` | O valor do campo de cabeçalho de resposta name. O argumento name deve ser convertido para letras minúsculas e os hífens devem ser convertidos para underscore. Por exemplo: `${sent_http_content_length}` assumirá o valor do cabeçalho Content-Length. | Response Phase |
+| `${sent_http_name}` | O valor do campo de cabeçalho de resposta name. O argumento name deve ser convertido para letras minúsculas e os hífens devem ser convertidos para underscore. Por exemplo: `${sent_http_content_length}` assumirá o valor do cabeçalho `Content-Length`. | Response Phase |
+| `${server_addr}` | O endereço IP do servidor que está recebendo a requisição HTTP. | Request Phase |
+| `${server_port}` | A porta HTTP do servidor que recebe a requisição. | Request Phase |
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/produtos/edge-application/rules-engine/index.mdx:110]: pode ser "que está recebendo", que nem o de cima?

**Review by hannah-sk — APPROVED** [review]

File: src/content/docs/en/pages/products/edge-application/rules-engine.mdx [hunk 5]
```diff
@@ -94,18 +94,22 @@ import TableContainer from "~/components/Table/TableContainer.astro"
...
 | `${request}` | Complete request query. | Request Phase<br />Response Phase |
 | `${request_method}` | The HTTP method of the request. For example: `GET`. | Request Phase<br />Response Phase |
 | `${request_uri}` | The complete URI of the request, with arguments (query string). | Request Phase<br />Response Phase |
 | `${scheme}` | The scheme of the request. For example: `http` or `https`. | Request Phase<br />Response Phase |
-| `${sent_http_name}` | The value of the response header field name. The name argument must be converted to lowercase and the hyphens must be converted to underscore. For example: `${sent_http_content_length}` will take the value of the Content-Length header. | Response Phase |
+| `${sent_http_name}` | The value of the response header field name. The name argument must be converted to lowercase and the hyphens must be converted to underscore. For example: `${sent_http_content_length}` will take the value of the `Content-Length` header. | Response Phase |
+| `${server_addr}` | The IP address of the server that is receiving the HTTP request. | Request Phase |
```
Comment (bru-andrade) [inline src/content/docs/en/pages/products/edge-application/rules-engine.mdx:109]:
```suggestion
| `${server_addr}` | The IP address of the server that will be receiving the HTTP request. | Request Phase |
```

File: src/content/docs/en/pages/products/edge-application/rules-engine.mdx (hunk 1 above)
Comment (bru-andrade) [inline src/content/docs/en/pages/products/edge-application/rules-engine.mdx:110]:
```suggestion
| `${server_port}` | The HTTP port of the server that will be receiving the request. | Request Phase |
```

File: src/content/docs/pt-br/pages/produtos/edge-application/rules-engine/index.mdx (hunk 4 above)
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/produtos/edge-application/rules-engine/index.mdx:110]:
```suggestion
| `${server_addr}` | O endereço IP do servidor que irá receber a requisição HTTP. | Request Phase |
| `${server_port}` | A porta HTTP do servidor que irá receber a requisição. | Request Phase |
```

**Pull request merged**